x-pack/plugins/ml/public/application/jobs/new_job/pages/new_job/wizard_horizontal_steps.tsx

```diff
@@ -79,6 +79,10 @@ export const WizardHorizontalSteps: FC<Props> = ({
   }
 
   function createStepProps(step: WIZARD_STEPS) {
+    let testSubj: string = '';
+    if (step === WIZARD_STEPS.TIME_RANGE) testSubj = 'mlJobWizardTimeRangeStep';
+    if (step === WIZARD_STEPS.JOB_DETAILS) testSubj = 'mlJobWizardJobDetailsStep';
+
```

Comment:
We can avoid these `if` conditions by simply adding `data-test-subj` to appropriate objects in `stepsConfig`.
```suggestion
```